src/main/java/nextstep/fp/Lambda.java

```diff
@@ -34,6 +35,16 @@ public static int sumAll(List<Integer> numbers) {
         return total;
     }
 
+    public static int sumAll(List<Integer> numbers, Conditional c) {
+        int total = numbers.stream()
+                .mapToInt(number -> number)
+                .sum();
+        if (c.test(total)) {
+            return total;
+        }
+        return sumAll(numbers);
```

Comment:
```suggestion
        return numbers.stream()        
                      .filter(c::test)
                      .sum();
```
이렇게 filter를 사용하면 된답니다!